_pages/es/index.md

```diff
@@ -27,7 +27,7 @@ Defend Internet Privacy and Policy (coming soon)
 
 {: #area-testpilot .area}
 [![image](/asserts/img/test-pilot.png)
-Use and test experimental Firefox features](/experiments/)
+Use and test experimental Firefox features](/es/experiments/)
```

Comment:
I don't know Jenkyll that well, but shouldn't this handle the language on its own so that change would not be necessary?